build/Common.props

```
@@ -33,7 +33,7 @@
     <MicrosoftPublicApiAnalyzersPkgVer>[3.3.3]</MicrosoftPublicApiAnalyzersPkgVer>
     <MicrosoftSourceLinkGitHubPkgVer>[1.1.1,2.0)</MicrosoftSourceLinkGitHubPkgVer>
     <OpenTelemetryCoreLatestVersion>[1.3.1,2.0)</OpenTelemetryCoreLatestVersion>
-    <OpenTelemetryCoreLatestPrereleaseVersion>[1.4.0-beta.2]</OpenTelemetryCoreLatestPrereleaseVersion>
+    <OpenTelemetryCoreLatestPrereleaseVersion>[1.4.0-beta.3,2.0)</OpenTelemetryCoreLatestPrereleaseVersion>
```

Based on additional changes (histogram API) I suppose that this range could be too optimistic, but we are just before release state, so I do not expect changes in used API in this projects.

Other option is
```suggestion
    <OpenTelemetryCoreLatestPrereleaseVersion>[1.4.0-beta.3]</OpenTelemetryCoreLatestPrereleaseVersion>
```

Please apply change if you find it better option.

---

For non-stable packages, we don't want to depend on a range.